Bottom: 16d6e3339ece757bcc8d807072013e5f3692c6ec
Top:    9f495f3d357be2d1afd6ad65f2763b15b3386f07
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 21:40:45 +0000

Refresh of ui_string.diff

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index b1b75fa..817c453 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -119,6 +119,7 @@ a zoom in/out.
 #include "set.h"
 #include "undo.h"
 #include "vendor.h"
+#include "free_atexit.h"
 
 #include "gui-icons-mode-buttons.data"
 #include "gui-icons-misc.data"
